BlueprintUICommonControls/Sources/AttributedLabel+Environment.swift

```diff
@@ -15,16 +15,25 @@ import UIKit
 ///
 public protocol URLHandler {
     func onTap(url: URL)
+    func isEquivalent(to other: URLHandler) -> Bool
 }
 
 struct DefaultURLHandler: URLHandler {
     func onTap(url: URL) {
         UIApplication.shared.open(url, options: [:], completionHandler: nil)
     }
+
+    func isEquivalent(to other: URLHandler) -> Bool {
+        true
```

Comment:
Do we need to ensure the type hasn't changed?
```suggestion
        other is Self
```

If not, why not just always return `true` in the `URLHandlerEnvironmentKey`'s `isEquivalent`?

Reply:
I don't it matters right? Given these URL handlers changing don't affect layout or measurement. Is there a case where we would want to remeasure if this handler changes?

Reply:
I suppose it shouldn't matter, but then when would it ever? Why have `isEquivalent` on the URLHandler protocol?

Reply:
Removing for now. If we bring back `appliesViewDescriptionIfNotEquivalent` we'll need some finessing, but we can figure that out then.